server/controllers/caseSummary/getCaseSection.test.ts

```diff
@@ -61,6 +62,40 @@ describe('getCaseSection', () => {
     )
   })
 
+  it('caches the contact history response in redis if CRN is not excluded or restricted', async () => {
```

Comment:
I might be missing something, but it doesn't look like this tests that the response is cached in Redis.

Reply:
wrong test name - fixed.